README.md

```diff
@@ -0,0 +1,105 @@
+# Change-impact-analysis Tool
```

Comment:
Not sure what editor you are using but this file could use a markdown-link run.

If you are using VS Code there is a markdown-lint extension that will fix most things for you.

Reply:
Okay! Just updated the readme.md after running markdown-link.